libraries/api/test/validate_test.js

```diff
@@ -129,6 +127,44 @@ suite(testing.suiteName(), function() {
     res.reply();
   });
 
+  builder.declare({
+    method: 'get',
+    route: '/test-double-json-send',
+    name: 'testDoubleJsonSend',
+    output: 'test-schema.yml',
+    category: 'API Library',
+    title: 'Test End-Point',
+    description: 'place to call to trigger a double send',
+  }, function(req, res) {
+    res.status(400).json({error: 'yep'});
+    res.status(200).reply({value: 1});
+  });
+
+  builder.declare({
+    method: 'get',
+    route: '/test-double-error-send',
+    name: 'testDoubleErrorSend',
+    output: 'test-schema.yml',
+    category: 'API Library',
+    title: 'Test End-Point',
+    description: 'place to call to trigger a double send',
+  }, function(req, res) {
+    res.reportError('InputError', 'uhoh', {});
+    res.status(200).reply({value: 1});
```

Comment:
`res.reportError` actually throws an exception, so in this case, the `res.status` line never runs -- which is why the "called send twice" error is never logged for this API method (but see the note below for why the test case appeared to pass).

If you switch the order of these two lines, it will make it a more useful test case.